code/datums/gamemodes/pod_wars.dm

```diff
@@ -3134,6 +3134,21 @@ proc/setup_pw_crate_lists()
 			else
 				return ..(target, start, user)
 
+	shoot_point_blank(mob/M, mob/user)
+		if (canshoot())
+			if (team_num)
+				if (team_num == 1 && user?.mind?.special_role == "NanoTrasen")
```

Comment:
This is good and how I was handling team checking at the beginning of development, but I made a proc to make this a bit more readable: `/proc/get_pod_wars_team_num(var/mob/user)`
Returns TEAM_NANOTRASEN, TEAM_SYNDICATE, or 0.

Reply:
fixed the point blank and the standard shooting, using the proc.